clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -2423,10 +2429,7 @@ LogicalResult cir::VTableAddrPointOp::verify() {
     return success();
 
   auto resultType = getAddr().getType();
-  auto intTy = cir::IntType::get(getContext(), 32, /*isSigned=*/false);
-  auto fnTy = cir::FuncType::get({}, intTy);
-
-  auto resTy = cir::PointerType::get(cir::PointerType::get(fnTy));
+  auto resTy = cir::PointerType::get(cir::VPtrType::get(getContext()));
 
   if (resultType != resTy)
```

Comment:
nit: I suggest to add `let cppFunctionName = "isPtrToVPtrType";` into `CIR_PtrToVPtr` constraint which can be used in places like this `if (isPtrToVPtrType(resultType))` without need to explicitly construct  `cir::PointerType::get(cir::VPtrType::get(getContext()));` before.

Reply:
I don't think this is even needed here, since the return type is now encoded as CIR_PtrToVPtr in the op definition, but I'll keep this tip in mind for future uses.